lib/engines/hcktest/tests.rb

```diff
@@ -447,7 +474,7 @@ def run(tests)
           queue_test(test, wait: true)
           list_tests
           handle_test_running
-
+          run_test_commands(test, :post_test_commands)
```

Comment:
It is weird to run `pre_tests_commands` just once for all runs while running `post_test_commands` for each run.